[2025-03-05T19:00:09.790+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-05T19:00:09.812+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.download_data scheduled__2025-03-05T18:00:00+00:00 [queued]>
[2025-03-05T19:00:09.822+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.download_data scheduled__2025-03-05T18:00:00+00:00 [queued]>
[2025-03-05T19:00:09.824+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-05T19:00:09.847+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): download_data> on 2025-03-05 18:00:00+00:00
[2025-03-05T19:00:09.856+0000] {standard_task_runner.py:72} INFO - Started process 5271 to run task
[2025-03-05T19:00:09.863+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline', 'download_data', 'scheduled__2025-03-05T18:00:00+00:00', '--job-id', '159', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpk981txse']
[2025-03-05T19:00:09.866+0000] {standard_task_runner.py:105} INFO - Job 159: Subtask download_data
[2025-03-05T19:00:09.900+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-05T19:00:09.943+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.download_data scheduled__2025-03-05T18:00:00+00:00 [running]> on host b5502d51568b
[2025-03-05T19:00:09.946+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-05T19:00:10.058+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='download_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-05T18:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-05T18:00:00+00:00'
[2025-03-05T19:00:10.061+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-05T19:00:10.091+0000] {data_pipeline.py:51} INFO - Executing script: /opt/airflow/scripts/data_acquisition.py
[2025-03-05T19:00:47.831+0000] {data_pipeline.py:53} INFO - Successfully executed: /opt/airflow/scripts/data_acquisition.py
[2025-03-05T19:00:47.832+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-05T19:00:47.842+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-05T19:00:47.843+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=download_data, run_id=scheduled__2025-03-05T18:00:00+00:00, execution_date=20250305T180000, start_date=20250305T190009, end_date=20250305T190047
[2025-03-05T19:00:47.917+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-05T19:00:47.960+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-05T19:00:47.968+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
